rostran/providers/terraform/c_template.py

```diff
@@ -23,7 +23,7 @@ def initialize(
         if os.path.isdir(path):
             for dirpath, dirnames, filenames in os.walk(path):
                 for filename in filenames:
-                    if filename.endswith((".tf", ".tftpl")):
+                    if filename.endswith(('.tf', '.tftpl', '.tfvars', '.metadata', '.mappings', '.conditions', '.rules')):
```

Comment:
please use `black` to format the project

Reply:
done